### ISSUE TITLE
Where are the expected timestamp values of Asset Tracker web app come from?

### ISSUE THREAD

**Issue by MLopezJ** [MLopezJ/LwM2M-Asset-Tracker#2]
# TL;DR

Asset tracker is expecting an already defined data format as the input in Azure. There are many timestamp values as part of it. Those timestamp values should come from **the device**. Those values should be send from the client (the device) using **resource from LwM2M Object**

# More

Asset Tracker web app is expected to receive an input with a defined data format, which is described [here](https://github.com/NordicSemiconductor/asset-tracker-cloud-docs/blob/saga/docs/cloud-protocol/state.reported.azure.json). 

There is an assuption that those values should come from the device, but the following question was asked just to confirm it:

```
Asset tracker is expecting an already defined data format as the input in Azure. There are many timestamp values as part of it.

Those timestamp values should come from: 
[ ] The server
[ ] The device
[ ] Is not relevant
```

**Comment by MLopezJ** [comment]
Those timestamp values should come from: 
[   ] The server
**[ X ] The device**
[  ] Is not relevant

**Comment by MLopezJ** [comment]
Those values should be send from the client using: 

1. resource from Object
2. Content-format 

**[X] 1**
[ ] 2
[ ] Both
[ ] Any of those
